pulpcore/pulpcore/app/serializers/__init__.py

```diff
@@ -7,7 +7,6 @@
     ModelSerializer,
     MasterModelSerializer,
     DetailIdentityField,
-    DetailRelatedField,
```

Comment:
One more thing, remove this one change please

Reply:
Why? This import was duplicated. I didn't remove the import, it was just duplicated.

Reply:
Ah, you're right.  For some reason (probably severe exhaustion) I was thinking it was an "unused import" or something.  Even though I saw that it was duplicated the day before.

Anyway... this looks good to merge.  Thanks @alexandrevicenzi